components/admin_console/admin_settings.tsx

```diff
@@ -28,8 +28,6 @@ export type BaseState = {
     errorTooltip: boolean;
 }
 
-type StateKeys = keyof BaseState;
```

Comment:
The previous way we had `typescript-eslint` configured was less picky about unused types. You'll also see this in a few places where I removed type parameters like in the Cypress type defintions.